src/style.css

```diff
@@ -1,3 +1,71 @@
-body {
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body,
+ul {
   margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  color: #000;
+  max-width: 1200px;
+  min-width: 1024px;
+  margin: 0 auto;
+  background-color: rgb(239, 243, 245);
+
+  --main-color: #00acdc;
+}
+
+/* Header styles */
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)